internal/core/graphics/brush.rs

```diff
@@ -53,6 +53,16 @@ impl Brush {
         }
     }
 
+    /// Returns true if this brush is a solid color
+    ///
+    /// ```
+    /// # use i_slint_core::graphics::*;
+    /// assert!(Brush::SolidColor(Color::from_argb_u8(0, 255, 128, 140)).is_solid_color());
+    /// ```
+    pub fn is_solid_color(&self) -> bool {
```

Comment:
Why do we need this function?

Reply:
It is not yet used. It's not problem to remove it.